benchmarks/Benchmark.Behaviors/Benchmark.cs

```diff
@@ -178,6 +181,10 @@ public void Setup()
 		_immediateHandler = _serviceProvider.GetRequiredService<SomeHandlerClass.Handler>();
 		_handler = _serviceProvider.GetRequiredService<DirectTimingBehavior>();
 		_request = new(Guid.NewGuid());
+
+		_abstractionServiceScope = _serviceProvider.CreateScope();
```

Comment:
should not need another scope. just the `_immediateHandlerAbstraction` field

Reply:
I did start off with that, but it causes the same behavior to be resolved for the regular as well as the abstracted handler - And since we set the inner handler in the Handlers constructor for both of them, the second one will run into an exception since the behaviours inner handler is already setup

Reply:
ah, makes sense. i'm good then. cost of doing business quicker on execution.